jupyterlab_bigquery/src/components/list_items_panel/list_tree_panel.tsx

```diff
@@ -305,10 +318,10 @@ class ListItemsPanel extends React.Component<Props, State> {
                 style={{ color: 'blue' }}
                 href="https://console.developers.google.com/apis/api/datacatalog.googleapis.com/overview"
               >
-                Google Data Catalog API.
+                Google Data Catalog API
               </a>{' '}
-              Once you click "Enable", this may take up to 2-3 minutes before
-              you can start searching.
+              for all pinned projects. Once you click "Enable", this may take up
```

Comment:
Maybe state `this may take several minutes` instead? Can we confidently promise an upper bound?

Reply:
This is the same message that is in API-enabling popups from the cloud console. Should I change it to several minutes or keep what is in the webUI to maintain consistency?

Reply:
We can leave it be for consistency